prisma/schema.prisma

```diff
@@ -28,7 +28,6 @@ model IABCategory {
   parentCollections Collection[]  @relation("CollectionIABParentCategory")
   childCollections  Collection[]  @relation("CollectionIABChildCategory")
 
-  @@unique([externalId])
```

Comment:
According to the docs we should use `@@unique` for a [compound unique field.](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#unique-1) [The v4 upgrade docs also enforce](https://www.prisma.io/docs/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-4#explicit-unique-constraints-on-one-to-one-relations) the use of `@unique` directive in line where the field defined.

Reply:
The upgrade doc also mentioned to remove the `references: []` from the `@relation("...", fields: [...], references: [id])`. 

But the prisma docs for `@relation` say other wise (`references` is required) 🙃. So I've left it as is for now and referred to shareable-list-api service which follows the same pattern for `@relation`.

Reply:
And regarding `@@unique` comment above, shareable-list-api still has the older pattern and seems like that still works with v4 too?

Reply:
The Shareable Lists API would have been set up on Prisma v.4 that probably pre-dated these additional schema changes? Unless we run the migration script there, too, we'll never know 🙃